src/current/v25.1/create-changefeed.md

```diff
@@ -23,6 +23,12 @@ The [examples](#examples) on this page provide the foundational syntax of the `C
 
 ### Legacy privilege model
 
+{{site.data.alerts.callout_info}}
+Starting in v22.2, CockroachDB introduces a new [system-level privilege model]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) that provides finer control over a user's privilege to work with the database, including creating and managing changefeeds.
```

Comment:
i don't understand this, this is referencing pretty far into the past to say "oh btw that was the legacy thing" but AFAICT this snippet of text cannot (meaningfully) retroactively accomplish this? even if it is "technically correct" etc? 

also if this statement is really true would not the new docs we're adding here about the new model also be ported back to those previous versions?  because otherwise how would this be a "legacy" model unless the new thing also existed in v22.2+ ?

whereas AFAICT in this PR the other docs about the non-legacy way of doing this are only being added to v25.1 ?

this feels "highly irregular", please enlighten me 🤓